Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Model/ModelLod.h

```diff
@@ -75,6 +75,7 @@ namespace AZ
                 StreamInfoList m_streamInfo;
 
                 ModelMaterialSlot::StableId m_materialSlotStableId = ModelMaterialSlot::InvalidStableId;
+                AZ::Name m_materialSlotName;
```

Comment:
I have not read through all of the changes yet to see how the name is being used. I have mentioned plans to replace the stable ID with the material name and use that in material assignment IDs. This would fix several material component workflow related issues, serve as both the ID and the label, and eliminate the need to search for material IDs before use in script. It would also fix a regression where material assignments used to be portable between models with the same material slot names. Before the stable ID, we used CRCs of the name as the sub ID and unique material ID. Need to plan a path to remove stable ID without breaking existing data.

Reply:
I'm currently only using this for debug output.

Reply:
That's fine. It will be useful for what I described above very soon